FSharp.AutoComplete/Program.fs

```diff
@@ -607,6 +611,21 @@ module internal Main =
 
             main state
 
+
+          | Methods ->
+            let meth = tyRes.GetMethods(line, col, lineStr)
```

Comment:
Maybe something to consider when deciding on a json format. Which parameter of which overload.